tokenserver/scripts/process_account_events.py

```diff
@@ -88,7 +88,15 @@ def process_account_event(config, body):
         body = json.loads(body)
         event = json.loads(body['Message'])
         event_type = event["event"]
-        email = event["uid"]
+        uid = event["uid"]
+        try:
+            email = "%s@%s" % (uid, event["iss"])
```

Comment:
So if ``event["iss"]`` doesn't exists it means we are using the previous API.

Can we use ``if "iss" in event`` instead or ``if '@' not in uid``?